MetaCocktailsSwiftData/Model/Components/Ingredients/IngredientType.swift

```diff
@@ -208,12 +209,13 @@ enum MeasurementUnit: String, Codable {
     case dash           = "dash"
     case dashes         = "dashes"
     case drops          = "drops"
-    case berries        = "berries"
-    case leaves         = "leaves"
+    case muddled        = "Muddled"
+    case gentlyMuddled  = "Gently muddled"
+    case nitroMuddled   = "Nitro muddled"
     case ml             = "ml"
-    case barspoon       = "Barspoon"
+    case barspoon       = "Bar-spoon"
```

Comment:
Barspoon is a relatively inaccurate measurement that I think we should move away from.
Since we only use barspoon in 5 recipes right now, can we change those recipes into teaspoon measurements instead?